src/components/common/Tooltip/tooltip.module.scss

```diff
@@ -18,14 +18,13 @@
     &::before {
         content: '';
         position: absolute;
-        transform: rotate(45deg);
         height: 12px;
-        bottom: -5px;
-        z-index: -1;
-        left: 8rem;
-        right: 8rem;
-        width: 13px;
+        width: 12px;
         background: $theme-primary;
+        bottom: -6px;
```

Comment:
why we are using negative value here?

Reply:
this is for that arrow to show below the tooltip 
![image](https://github.com/user-attachments/assets/75d3ff17-8c2b-46f3-8a64-9ded5ecaf550)

Reply:
can we do that without the negative value

Reply:
done it without using negative values